packages/kbn-es-query/src/es_query/index.d.ts

```diff
@@ -0,0 +1,31 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+export function buildQueryFromFilters(filters: unknown[], indexPattern: unknown): unknown;
+export function buildEsQuery(
+  indexPattern: unknown,
+  queries: unknown,
+  filters: unknown,
+  config?: {
+    allowLeadingWildcards: boolean;
+    queryStringOptions: unknown;
```

Comment:
Will queryStringOptions not be an object type? The [default is `{}`](https://github.com/elastic/kibana/blob/master/packages/kbn-es-query/src/es_query/build_es_query.js#L39).